audio_service/ios/Classes/AudioServicePlugin.h

```diff
@@ -36,9 +36,9 @@ enum MediaAction {
     APrepareFromUri,
     ASetRepeatMode,
     AUnused_1, // deprecated (setShuffleModeEnabled)
-    AUnused_2, // setCaptioningEnabled
+    ASetCaptioningEnabled,
     ASetShuffleMode,
-    // Non-standard
```

Comment:
Can week keep this comment or something of the sort?

Reply:
I'm not sure what does it mean?

Reply:
Maybe a better comment is: iOS-specific. I've put a comment in the platform interface at the same point in the enum.